client/components/jetpack/threat-description/index.tsx

```diff
@@ -24,11 +24,34 @@ export interface Props {
 	context?: object;
 	diff?: string;
 	filename?: string;
+	isFixable: bool;
 }
 
 class ThreatDescription extends React.PureComponent< Props > {
 	renderTextOrNode( content: string | TranslateResult | ReactNode ) {
-		return <p className="threat-description__section-text">{ content }</p>;
+		return <>{ content }</>;
+	}
+
+	renderFixTitle() {
+		const { status, isFixable } = this.props;
+
+		switch ( status ) {
+			case 'fixed':
+				return translate( 'How did Jetpack fix it?' );
+				break;
+
+			case 'current':
+				if ( isFixable ) {
+					return translate( 'How will we fix it?' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'How we will fix it?' )` *ES Score: 7*